### DISCUSSION TITLE
How to copy part of the text from screen?

### DISCUSSION THREAD

**Discussion by Petri-X** [microsoft/terminal#16397] [Q&A] [answered]
Hi,
In earlier versions of command prompt and PowerShell I have used plenty of area copy:
![image](https://github.com/microsoft/terminal/assets/43988151/351ef2fb-7526-430a-895c-59bfdffcf0f5)

But the terminal forced the selection now to whole row:
![image](https://github.com/microsoft/terminal/assets/43988151/d3eccb5b-2da6-4b35-8e7f-6afbe841357c)

How I should be able to fix this?

**Comment by zadjii-msft** [comment, marked as answer]
Hold <kbd>alt</kbd> while making the selection 😉

**Comment by Petri-X** [comment]
@zadjii-msft 

You are my hero !!!!!!!

Big thanks!


...I Mean HUGE BIG THANKS!!!
